build/build.proj

```diff
@@ -17,7 +17,7 @@
     <SolutionFile Include="$(RepositoryRootDirectory)sdk-templates.sln" Condition="'$(BuildTemplates)' == 'true'" />
 
     <SigningProjectFile Include="$(RepositoryRootDirectory)build\Signing\Microsoft.NET.Signing.proj"/>
-    <NuGetProjectFile Include="$(RepositoryRootDirectory)build\Nuget\Microsoft.NET.Nuget.proj" />
+    <NuGetProjectFile Include="$(RepositoryRootDirectory)build\Nuget\Microsoft.NET.Nuget.csproj" />
```

Comment:
Weird. Yes, I think you can just set <PackProjectInputFile>$(MSbuildProjectFullPath)</PackProjectInputFile> yourself.